lib/cdo/rack/upgrade_insecure_requests.rb

```diff
@@ -69,7 +69,8 @@ def call(env)
           headers['X-Frame-Options'] = ''
         end
 
-        unless policies.empty?
+        unless policies.empty? || headers.key?('Content-Security-Policy')
+          puts "applying content security policy"
```

Comment:
i dont know that it's common for us to puts strings like this. not sure if we have a better logging mechanism